types/recursive-readdir/index.d.ts

```diff
@@ -1,20 +1,21 @@
-// Type definitions for recursive-readdir v1.2.1
+// Type definitions for recursive-readdir 2.2
```

Comment:
since this is a major version upgrade please create a new folder v1 and copy the old files before making the change. please see https://github.com/DefinitelyTyped/DefinitelyTyped#i-want-to-update-a-package-to-a-new-major-version

Reply:
Fixed.